### ISSUE TITLE
add filter to TE2 wpsc_get_customer_account_tabs

### ISSUE THREAD

**Issue by curtismchale** [wp-e-commerce/WP-e-Commerce#1820]
Currently I see no way to add a tab to the customer account.

I'd like to see this added just after the current $tabs array so it would look like...

```php
	$tabs = array(
		'orders'          => _x( 'Orders', 'customer account tab', 'wpsc' ),
		'digital-content' => _x( 'Digital Contents', 'customer account tab', 'wpsc' ),
		'settings'        => _x( 'Settings', 'customer account tab', 'wpsc' )
	);

	$tabs = apply_filters( 'wpsc_user_profile_tabs', $tabs );
```

This actually maintains backwards compatibly for tabs used in TE1 so a tab would...just work in TE1 and TE2

**Comment by JustinSainton** [comment]
@curtismchale As much as I like the idea of using the same filters for the tabs, I think there could be quite a bit of unexpected output in TEv2 if we did that.  Plugins that are not compatible with TEv2 would be showing tabs here without necessarily having the matching controllers and views for it.  I'd rather it be opt-in, separate from v1, so that we know anything showing up there is done intentionally.

That said, spot on, we do in fact need a filter here.

**Comment by JustinSainton** [comment]
Closed via e448759
